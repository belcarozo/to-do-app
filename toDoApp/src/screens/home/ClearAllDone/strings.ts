export const strings = {
  clear: 'CLEAR ALL DONE',
}
