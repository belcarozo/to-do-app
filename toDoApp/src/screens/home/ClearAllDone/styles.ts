import { StyleSheet } from 'react-native'
import { Color } from '../../../styles/Pallete'

export const styles = StyleSheet.create({
  title: {
    fontSize: 14,
    paddingVertical: 9,
    color: Color.xlPink,
    //fontFamily: 'SourceSansPro',
    fontWeight: 'bold',
  },
})
