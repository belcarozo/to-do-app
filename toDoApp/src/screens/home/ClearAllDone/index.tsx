import React from 'react'
import { styles } from './styles'
import { Pressable, Text } from 'react-native'

export const ClearAllDone: React.FC<{ clearDone: () => void }> = ({
  clearDone,
}) => (
  <Pressable onPress={clearDone}>
    <Text style={styles.title}>CLEAR ALL DONE</Text>
  </Pressable>
)
