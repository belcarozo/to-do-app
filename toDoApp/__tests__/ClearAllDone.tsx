import React from 'react'
import { shallow } from 'enzyme'
import { ClearAllDone } from '../src/screens/Home/ClearAllDone'

describe('Rendering new task', () => {
  it('should match to snapshot', () => {
    const component = shallow(<ClearAllDone clearDone={jest.fn()} />)
    expect(component).toMatchSnapshot()
  })
})
