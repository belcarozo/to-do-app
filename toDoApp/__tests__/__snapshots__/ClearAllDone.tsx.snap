// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rendering new task should match to snapshot 1`] = `
<Pressable
  onPress={[MockFunction]}
>
  <Text
    style={
      Object {
        "color": "rgb(255, 25, 123)",
        "fontSize": 14,
        "fontWeight": "bold",
        "paddingVertical": 9,
      }
    }
  >
    CLEAR ALL DONE
  </Text>
</Pressable>
`;
